src/main/java/core/basesyntax/service/impl/FruitTransactionMapper.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.enums.Operation;
+import core.basesyntax.service.Mapper;
+
+public class FruitTransactionMapper implements Mapper<FruitTransaction, String> {
```

Comment:
The class `FruitTransactionMapper` should not begin with an empty line. According to the checklist, don't begin class or method implementation with an empty line.